tools/@aws-cdk/node-bundle/src/notice.ts

```diff
@@ -0,0 +1,264 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import type { Dependency, Attribution, NoticeViolations } from './model';
+import { shell } from './shell';
+
+/**
+ * Valid licenses that are ok to redistribute.
+ */
+const DEFAULT_VALID_LICENSES = [
```

Comment:
I'd call this `ALLOWED_LICENSES` or `LICENSE_ALLOW_LIST`. Valid has connotations...

Reply:
Of course i initially used `ALLOWED` and switched...